RoddyCore/src/de/dkfz/roddy/execution/jobs/Job.groovy

```diff
@@ -158,9 +158,9 @@ class Job extends BEJob<BEJob, BEJobResult> {
                 , getResourceSetFromConfiguration(toolID, context)
                 , []
                 , [:]
-                , Roddy.getJobManager())
+                , Roddy.getJobManager()
+                , JobLog.toOneFile(context.loggingDirectory))
```

Comment:
How does toOneFile work? What does it do? Can you make the method name more descriptive? Above you work with outFile.

Reply:
It means that the job log is written to one file, as opposed to the method toSeparateStdoutAndError